UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -31,6 +31,7 @@
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.4.8",
     "com.unity.ugui": "1.0.0",
+    "com.unity.xr.interaction.toolkit": "2.2.0",
```

Comment:
Should this be updated in our package dependencies instead of just the sample project?

Reply:
Pushed!